dataBase/DBCHERRY.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python 
+import sqlite3 as sql 
```

Comment:
we need it, please keep it

On Wed, Jun 8, 2016 at 7:54 PM, harryhk <notifications@github.com> wrote:

> In dataBase/DBCHERRY.py
> <https://github.com/harryhk/rsData/pull/2#discussion_r66362476>:
>
> > @@ -0,0 +1,43 @@
> > +#!/usr/bin/env python
> > +import sqlite3 as sql
>
> we don't need this file anymore. please remove it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/harryhk/rsData/pull/2/files/7eb4b28818a1cfdd970453ccdd85a7b6610100f3#r66362476>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ6NgS0tyw7J0QX-MOaNT8BLvq17ls06ks5qJ1ZOgaJpZM4IwdnK>
> .
>